module/platform/darwin/HardwareSimulator/README.md

```diff
@@ -1,40 +1,36 @@
-Darwin Hardware I/O
+NUgus Hardware I/O
 ===================
 
 ## Description
 
-This module is responsible for communicating with the Darwin robot's CM740
-controller.
+This module is responsible for communicating with the NUgus' CM740 controller.
```

Comment:
If we're updating this might as well fix it so it's not a copy of the HardwareIO readme. This module doesn't connect to the CM740 (or any device), instead, it simulates all the sensor info we want.

Reply:
Also, see my suggestion for the README in this PR

Reply:
> Also, see my suggestion for the README in this PR
was this meant to be a link?

Reply:
@Bidski could you please clarify what you meant

Reply:
@KipHamiltons look at the PR review comments I made the HardwareIO README.md file

Reply:
Ahh I understand. thanks